src/System.Security.Cryptography.Cng/src/Internal/Cryptography/Helpers.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Security.Cryptography;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+using ErrorCode = Interop.NCrypt.ErrorCode;
+
+namespace Internal.Cryptography
+{
+    internal static class Helpers
+    {
+        public static byte[] CloneByteArray(this byte[] src)
```

Comment:
Doesn't need to be as part of this PR, but we should really look into consolidating a bunch of these Helpers.  For example, I think I've seen this Clone method duplicated in at least a handful of the other crypto libs.